taesch/README.md

```diff
@@ -1,16 +1,54 @@
 # taesch
+Komfortables einkaufen
 
-A new Flutter project.
+<img src="https://github.com/sorrowcode/taesch-/blob/developer/taesch/assets/logo.png?raw=true" width=20% height=20%>
+
+taesch mach das Einkaufen einfacher. taesch lernt aus ihrem Einkaufsverhalten in einem Supermarkt und giebt tipps für die reihenfolge.
 
 ## Getting Started
+Vorbedingung
+- [git](https://git-scm.com/)
+- [Flutter SDK](https://docs.flutter.dev/development/tools/sdk/releases)
+- [Docker](https://www.docker.com/)
+- Apple Mac Gerät (für IOS)
+
+Zur Entwickung in Flutter eignet sich gut [Android Studio](https://developer.android.com/studio)
+
+
+Klonen des Projektes
+```bash
+  git clone https://github.com/sorrowcode/taesch-
+```
+
+Code ordner betreten
+```bash
+  cd taesch-/taesch
+```
+
+Installieren der Abhängigkeiten
+```bash
+  flutter pub get
+```
+
+Ein debug Paket erstellen
+```bash
+  flutter run
+```
+
+Ein Auslieferungspaket erstellen
+```bash
+  flutter run --release
+```
 
```

Comment:
es können auch die Tests über die Flutter-Konsole ausgeführt werden, aber damit das in der CI funktioniert, ist ein Docker-Image notwendig.
-> `flutter test`